src/factory/load_gltf.rs

```diff
@@ -269,22 +276,106 @@ impl super::Factory {
+        let mut clips = Vec::new();
+        for animation in gltf.animations() {
+            let mut tracks = Vec::new();
+            let name = animation.name().map(str::to_string);
+            for channel in animation.channels() {
+                let sampler = channel.sampler();
+                let target = channel.target();
+                let node = target.node();
+                let object = match node_map.get(&node.index()) {
+                    Some(object) => object.clone(),
+                    // This animation does not correspond to any loaded node.
+                    None => continue,
+                };
+                let input = sampler.input();
+                let output = sampler.output();
+                use gltf::animation::InterpolationAlgorithm::*;
+                let interpolation = match sampler.interpolation() {
+                    Linear => animation::Interpolation::Linear,
+                    Step => animation::Interpolation::Discrete,
+                    CubicSpline => animation::Interpolation::Cubic,
+                    CatmullRomSpline => animation::Interpolation::CatmullRom,
+                };
+                use animation::{Binding, Track, Values};
+                let times: Vec<f32> = AccessorIter::new(input, buffers).collect();
+                let (binding, values) = match target.path() {
+                    gltf::animation::TrsProperty::Translation => {
+                        let values = AccessorIter::<[f32; 3]>::new(output, buffers)
+                            .map(|v| {
+                                mint::Vector3 {
+                                    x: v[0],
+                                    y: v[1],
+                                    z: v[2],
+                                }
+                            })
+                            .collect::<Vec<_>>();
+                        assert_eq!(values.len(), times.len());
+                        (Binding::Position, Values::Vector3(values))
+                    }
+                    gltf::animation::TrsProperty::Rotation => {
+                        let values = AccessorIter::<[f32; 4]>::new(output, buffers)
+                            .map(|r| {
+                                mint::Quaternion {
+                                    s: r[3],
+                                    v: mint::Vector3 {
+                                        x: r[0],
+                                        y: r[1],
+                                        z: r[2],
+                                    },
+                                }
+                            })
+                            .collect::<Vec<_>>();
+                        assert_eq!(values.len(), times.len());
+                        (Binding::Orientation, Values::Quaternion(values))
+                    }
+                    gltf::animation::TrsProperty::Scale => {
+                        let values = AccessorIter::<[f32; 3]>::new(output, buffers)
```

Comment:
needs a warning that non-uniform scales are mishandled

Reply:
Does that mean a `warn!` log or a note in the docs (or both)?

Reply:
I'm afraid a `warn` repeated thousands of time would be troublesome. Just a comment would be fine for now.